### DISCUSSION TITLE
[DICA] -  Fundamentos do Azure - **ERRO: 403 ao publicar o pacote.**

### DISCUSSION THREAD

**Discussion by EdsonSantosActuar** [balta-io/forum#123] [Tira Dúvidas] [answered]
Fala devs, passando aqui para compartilhar uma dificuldade que tive e que consegui resolver na hora de subir o pacote.

Ao nomear o **PackageId** coloque nome único, caso não seja você recebera  o erro **403 - The specified API key is invalid, has expired, or does not have permission to access the specified package**.

Para resolver nomeei o **PackageId** com o seguinte padrão **GUID+MoneyExtension**.  `<PackageId>cb6855e0691b4136ac7f11b4b4228288MoneyExtension</PackageId>`

Espero ajudar alguém que esteja passando por essa dificuldade!

**Comment by andrebaltieri** [comment, marked as answer]
@EdsonSantosActuar muito obrigado pela ajuda 💜

**Comment by jose-vitor-quadrado** [comment]
No meu caso para conseguir publicar editei o `Glob pattern` para `*`

**Reply by andrebaltieri** [reply]
💜

**Comment by ricardoo-cavalheiro** [comment]
Me ajudou aqui. Só lembrando que depois de editar o `<PackageId></PackageId>` é necessário executar novamente o `build` e o `pack` e rodar o comando `dotnet nuget push` apontando para o novo `PackageId`.
